CMakeLists.txt

```diff
@@ -472,7 +472,7 @@ ELSE()
 ENDIF(WANT_DEBUG_FPE)
 
 # check for libsamplerate
-PKG_CHECK_MODULES(SAMPLERATE REQUIRED samplerate>=0.1.8)
+PKG_CHECK_MODULES(SAMPLERATE REQUIRED samplerate>=0.1.9)
```

Comment:
FYI, Ubuntu 16.04 still ships with `libsamplerate` 0.1.8. To avoid the compile issue with old `libsamplerate`, you can use `const_cast` instead. I've looked into the code of the library, and it looks safe.

Reply:
Oky doky :D I'll do that.